jobs/build/coreos-installer_sync/extract.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -uo pipefail
```

Comment:
`rpm2cpio "${rpm}" | zstd -d | cpio -idm --quiet ./usr/bin/coreos-installer` will return 1 without message but content extracted successfully, so remove -e to not break